php-frontend/src/main/java/org/sonar/php/tree/symbols/SymbolVisitor.java

```diff
@@ -173,6 +185,10 @@ private void createMemberSymbols(ClassTree tree) {
         ClassPropertyDeclarationTree classPropertyDeclaration = (ClassPropertyDeclarationTree) member;
         for (VariableDeclarationTree field : classPropertyDeclaration.declarations()) {
           createSymbol(field.identifier(), Symbol.Kind.FIELD).addModifiers(classPropertyDeclaration.modifierTokens());
+          ExpressionTree initValue = field.initValue();
+          if(initValue != null) {
+            initValue.accept(this);
```

Comment:
I don't understand why we need that as we should visit the class members anyway.

Reply:
Ok, that's because of `visitClassPropertyDeclaration` which is overridden in `SymbolVisitor`.